### DISCUSSION TITLE
Example output of styled layouts

### DISCUSSION THREAD

**Discussion by unscripted** [whitecube/nova-flexible-content#315] [Q&A]
I'm fairly new to Laravel and was happy to find this package as I'm very familiar with ACF and WordPress.  I have been able to build the flexible content fields for all of my models but I'm struggling to understand how to associate a specific "design" with each layout type.

My goal is to have a view partial for each layout type that includes the design for the output of each field. When on a page, the partial would be used for each instance of the layout.

If someone could point me to a repo or site that has similar functionality so that I could have a reference, that would be a huge help.

Thank you in advance!

**Comment by voidgraphics** [comment]
I don't have a proper example on hand for you, but I remember the way I've done this before.

Assuming you have a `views/flexible` directory with one blade file for each layout in your flexible field, if each blade file is named exactly like the `key` of each layout, you can just loop over the layouts and include the view corresponding to the current loop item's key

Pseudo-code:
```blade
@foreach($layouts as $layout)
    @include('flexible.' . $layout->key())
@endforeach
```
